### DISCUSSION TITLE
Add Tailwind Css

### DISCUSSION THREAD

**Discussion by Publisher5047** [payloadcms/payload#2240] [Feature Requests & Ideas]
Tailwind is very popular and easy to custom, edit. I just create topic here. I will contribute to rebuild core component if this accept.

**Comment by tyteen4a03** [comment]
I also think utility classes would help a lot in standardising Payload's design system.

**Comment by revnelson** [comment]
Check out this thread: https://github.com/payloadcms/payload/discussions/845

**Comment by GermanJablo** [comment]
If the proposal is to use Tailwind, that's already possible and should be easy, especially since we added scopes to our styles.

If the proposal is to migrate our styles from Payload to Tailwind: unfortunately, it's not in our plans, as much as I'd like it to be.

Reasons:
- Migration is costly. Even if someone from the community did it, the review would be complex.
- We'd have to convince the entire team.
- We'd have to maintain some semantic classes to avoid breaking selectors.

**Reply by tyteen4a03** [reply]
Since we're ditching SCSS in 4.0, the work would need to be done anyway?

**Reply by GermanJablo** [reply]
As far as I know, the plan is to migrate from SCSS to CSS, not to Tailwind, which is much easier.
